src/detectors/Golang/GolangLanguageDetector.ts

```diff
@@ -16,19 +17,21 @@ export class GolangLanguageDetector implements ILanguageDetector {
 
   async detectLanguage(): Promise<LanguageAtPath[]> {
     const result: LanguageAtPath[] = [];
-    const packageFiles = await this.fileInspector.scanFor(fileNameRegExp('Gopkg.toml'), '/');
+    let packageFiles = await this.fileInspector.scanFor(fileNameRegExp('go.mod'), '/');
```

Comment:
`go.mod` is preferred nowadays, though there may be projects that still use older dependency tools, I have only implemented support for go modules